services/Service.py

```diff
@@ -112,6 +151,16 @@ def convertTagKeyTagValueIntoKeyValue(self, tags):
             nTags.append({'Key': i['TagKey'], 'Value': i['TagValue']})
             
         return nTags
+    
+    def processChartData(self):
+        if len(self.chartData) > 0:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0`

[Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)